pipeline/transformer/transformer_calm/src/main/scala/uk/ac/wellcome/platform/transformer/calm/CalmTransformer.scala

```diff
@@ -34,6 +34,16 @@ object CalmTransformer extends Transformer[CalmRecord] with CalmOps {
       )
     }
 
+  def shouldTransform(record: CalmRecord): Boolean =
+    record
+      .get("Transmission")
+      .map(_.toLowerCase)
+      .map {
+        case "no" => false
+        case _    => true
+      }
```

Comment:
I assume the other value you'd expect here would be `"yes"`?

Suggestions:

* Add a comment explaining what this field means in CALM (what have you been told it means?)
* Add a third branch like so:

```suggestion
      .map {
        case "no"  => false
        case "yes" => true
        case other =>
          info(s"Unrecognised value for Transmission field; assuming true: $other")
          true
      }
```

So if we start getting works appearing that we don't expect, we can see what happened here.

(I can imagine, for example, a record with value `"no "` that this would let through, because CALM does no checking of values.)